### PR TITLE
Fix styling and bug

### LINKED ISSUE #6433: Lab requisition when orders are deleted
### Describe the bug
The requisition form stores the ID of the lab on the first opening of the form. If the lab order is deleted from the system. The requisition form keeps pulling that same number. Because there is no correlation between the two tables.  

### To Reproduce

1. Create an order.
2. Create a requisition form
3. Delete the order
4. Open requisition the form.
5. See the same number that has been deleted

### Expected behavior
After the order is deleted, the requisition system should delete the order also. 

### Client configuration

- Browser:
- OpenEMR version:
- Operating system:

### PR DESCRIPTION
**Pull request by juggernautsei**
Fixes #6433

#### Short description of what this resolves:
Fixes bugs in the order deletion and the requisition deletion. 
Updates styling to bootstrap x

### CODE REVIEW

File: library/lab.inc.php [hunk 1]
```diff
@@ -84,15 +84,15 @@ function getSelfPay($pid)
...
+    $sql = "SELECT `npi`, `upin` FROM `users` WHERE `id` = ?";
     $res = sqlQuery($sql, array($prov_id));
     return array($res['npi'], $res['upin']);
 }
 
 /**
  * @return array
  */
-function getProcedureProvider($prov_id)
+function getProcedureProvider($prov_id): array
```
Comment (adunsulag) [inline library/lab.inc.php:95]: sqlFetchArray can return an array or false.  Should probably set line 101 to be $labs = sqlFetchArray($res) ?? []; or change the return type of this function.

**Review by adunsulag — CHANGES_REQUESTED** [review]
Noted a couple minor things here.  Nothing major.

File: interface/forms/requisition/new.php [hunk 2]
```diff
@@ -64,7 +57,7 @@
 ?>
 
 <!DOCTYPE html>
-<html>
+<html lang="en">
```
[the remark this answers is not in the document]
Comment (juggernautsei) [reply interface/forms/requisition/new.php:60]: That is correct IDE keeps doing that I will change the settings.

**Comment by juggernautsei** [comment]
staying alive

File: interface/forms/requisition/new.php [hunk 3]
```diff
@@ -136,64 +129,72 @@
...
+    <?php
+    if (empty($order)) {
+            echo "<div class='text-center mt-5'><span>" .
+                xlt('procedure order not found in database contact tech support') . "</span></div>";
+            exit;
+    }
+    if (empty($oid)) {
+            print "<div class='text-center mt-5'><span>" .
+         xlt('No Order found, please enter procedure order first') . "</span></div>";
+            exit;
```
[the remark this answers is not in the document]
Comment (adunsulag) [reply interface/forms/requisition/new.php:148]: Still missing the </body></html> tag on 142 and 147 and then we can bring this in.  Sorry for the delay on the turnaround here.

File: interface/forms/requisition/new.php [hunk 4]
```diff
@@ -136,64 +129,72 @@
       float: left;
 
   }
-  .barcode{
-      position: relative;
-      right: -380px;
-  }
+
 </style>
+    <title><?php echo xlt('Lab Requisition ') . $lab; ?></title>
```
Comment (bradymiller) [inline interface/forms/requisition/new.php:134]: need to escape $lab via `text($lab)`

File: interface/forms/requisition/new.php (hunk 3 above)
[the remark this answers is not in the document]
Comment (bradymiller) [reply interface/forms/requisition/new.php:148]: hi @juggernautsei , looks like still need to address above. Then this should be ready.

[the remark this answers is not in the document]
Comment (juggernautsei) [reply interface/forms/requisition/new.php:148]: I am not sure why these are not being marked outdated. I have made an additional changed sugged by the IDE.

**Comment by juggernautsei** [comment]
Bump

File: interface/forms/requisition/new.php [hunk 5]
```diff
@@ -136,64 +129,72 @@
       float: left;
 
   }
-  .barcode{
-      position: relative;
-      right: -380px;
-  }
+
 </style>
+    <title><?php echo xlt('Lab Requisition ') . text($lab); ?></title>
```
Comment (bradymiller) [inline interface/forms/requisition/new.php:134]: sorry, just noted this. Need to remove the trailing space in the translation constant:
```
<title><?php echo xlt('Lab Requisition') . ' ' . text($lab); ?></title>
```

**Comment by bradymiller** [comment]
@juggernautsei , code looks good and bringing in. thanks for the improvements!

**Pull request merged**